front-end/studio/src/app/pages/apis/{apiId}/editor/_components/forms/path/operation/info-section.component.html

```diff
@@ -32,6 +32,14 @@
                                        [baseNode]="operation" nodePath="operationId"
                                        (onChange)="changeOperationId($event)"></pf-inline-text-editor>
             </div>
+
+            <!-- Deprecated -->
+            <div class="section-field deprecated">
+                <drop-down [value]="operation.deprecated"
+                           [options]="deprecatedOptions()"
+                           (onValueChange)="changeDeprecated($event)"
+                           [noSelectionLabel]="'not-deprecated'"></drop-down>
```

Comment:
Can you change this to:

`noSelectionLabel="Not Deprecated"`

Reply:
In other words, you can drop the braces around `noSelectionLabel` and then just have the value in quotes (without the single quotes).  And then the value should match what you have in the dropdown options later on. :)